kotlin/samples/todo-android/app/build.gradle

```diff
@@ -26,6 +26,12 @@ android {
     targetSdkVersion versions.targetSdk
     versionCode 1
     versionName "1.0.0"
+
+    testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
```

Comment:
If you think this will be the same for all apps you could just extract it into a separate `.gradle` file and apply to demo apps.

Something like:

./buildscript/configure-sample-app.gradle
```
apply plugin: 'com.android.application'

android {
  defaultConfig {
    testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
  }

  testOptions {
    animationsDisabled = true
  }
}

dependencies {
  androidTestImplementation deps.test.androidx.espresso.core
  androidTestImplementation deps.test.androidx.junitExt
  androidTestImplementation deps.test.androidx.uiautomator
}
```

and just apply that at the top:
```
apply from: rootProject.file('.buildscript/configure-sample-app.gradle')
apply plugin: 'kotlin-android'

android rootProject.ext.defaultAndroidConfig

android {
  defaultConfig {
......
```